tests/unit/test_dojson_utils.py

```diff
@@ -0,0 +1,182 @@
+            <subfield code="6">880-04</subfield>
+            <subfield code="a">Esakov, V.D</subfield>
+        </datafield>
+        <datafield tag="710" ind1="2" ind2=" ">
+            <subfield code="6">880-05</subfield>
+            <subfield code="a">Rossiĭskai︠a︡ akademii︠a︡ nauk.</subfield>
+            <subfield code="b">Institut rossiĭskoĭ istorii</subfield>
+        </datafield>
+        <datafield tag="880" ind1="0" ind2="0">
+            <subfield code="6">245-01/(N</subfield>
+            <subfield code="a">Академия Наук в решениях</subfield>
+            <subfield code="b">1922-1991 /</subfield>
+            <subfield code="c">Российская Академия Наук</subfield>
+        </datafield>
+        <datafield tag="880" ind1="0" ind2=" ">
+            <subfield code="6">505-03/(N</subfield>
+            <subfield code="a">[Т. 1]: Академия Наук</subfield>
+        </datafield>
+        <datafield tag="880" ind1="1" ind2=" ">
+            <subfield code="6">700-04/(N</subfield>
+            <subfield code="a">Есаков, В.Д</subfield>
+        </datafield>
+        <datafield tag="880" ind1="2" ind2=" ">
+            <subfield code="6">710-05/(N</subfield>
+            <subfield code="a">Российская академия наук.</subfield>
+            <subfield code="b">Институт российской истории</subfield>
+        </datafield>
+        <datafield tag="880" ind1=" " ind2="1">
+            <subfield code="6">264-02/(N</subfield>
+            <subfield code="a">Москва :</subfield>
+            <subfield code="b">РОССПЭН,</subfield>
+            <subfield code="c">2000-&gt;</subfield>
+        </datafield>
+        <datafield tag="900" ind1=" " ind2=" ">
+            <subfield code="a">goala</subfield>
+        </datafield>
+        <datafield tag="902" ind1=" " ind2=" ">
+            <subfield code="a">geulru</subfield>
+            <subfield code="b">2002/05</subfield>
+        </datafield>
+        <datafield tag="919" ind1=" " ind2=" ">
+            <subfield code="9">610 27</subfield>
+            <subfield code="a">Académie des sciences de l&apos;URSS</subfield>
+            <subfield code="x">Parti communiste de l&apos</subfield>
+            <subfield code="y">1922 / 1991</subfield>
+            <subfield code="2">chrero</subfield>
+        </datafield>
+        <datafield tag="957" ind1=" " ind2=" ">
+            <subfield code="a">geulru</subfield>
+        </datafield>
+        <datafield tag="982" ind1=" " ind2=" ">
+            <subfield code="2">ge-ulru</subfield>
+            <subfield code="a">1A aka/nau</subfield>
+        </datafield>
+        <datafield tag="992" ind1=" " ind2=" ">
+            <subfield code="a">BFLA 148817</subfield>
+            <subfield code="x">ge/ulru/a/02/195</subfield>
+        </datafield>
+    </record>
+    """
+    rero_overdo = ReroIlsMarc21Overdo()
+    marc21json = create_record(marc21xml)
+    data = rero_overdo.do(marc21json)
+    # assert data.get('type') == 'book'
```

Comment:
What do you test here? No assert!

Reply:
Deleted file tests/unit/test_dojson_utils.py